charts/wire-server/requirements.yaml

```diff
@@ -58,7 +58,7 @@ dependencies:
     - haskellServices
     - services
 - name: brig
-  version: "0.1.2"
+  version: "0.1.3"
```

Comment:
let's perhaps keep all versions for all wire-server charts in sync, run `set-version.sh wire-server 0.1.3` - otherwise this change won't be published.